test/module/shared_model/backend_proto/shared_proto_transaction_test.cpp

```diff
@@ -52,12 +55,22 @@ TEST(ProtoTransaction, Builder) {
   command.mutable_amount()->mutable_value()->set_fourth(1000);
   command.mutable_amount()->set_precision(2);
 
-  auto tx = shared_model::proto::TransactionBuilder()
-                .txCounter(tx_counter)
-                .creatorAccountId(account_id)
-                .addAssetQuantity(account_id, asset_id, amount)
-                .build();
-  auto &proto = tx.getTransport();
+  auto keypair =
+      shared_model::crypto::CryptoProviderEd25519Sha3::generateKeypair();
+  auto signedProto = shared_model::crypto::CryptoSigner<>::sign(
```

Comment:
Split this test to signed & unsigned protos

Reply:
Unsigned protos are "illegal" and should not be available via API so test for unsigned will be pretty useless.